scripts/gha/summarize_test_results.py

```diff
@@ -75,6 +75,7 @@
 
 CAPITALIZATIONS = {
     "macos": "MacOS",
+    "macosarm": "MacOS-ARM",
```

Comment:
I prefer to have only one MacOS, and make arch as a secondary description. Some more extra work is required here for the result summary report. I'll figure it out.

Reply:
Do you want to work in this same branch?